lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
That's a tough one to decide. I feel like a separate component makes more sense, but it also seems like unnecessary weight.